src/screens/admin/AddFields.tsx

```
@@ -0,0 +1,475 @@
+      setIsAdding(false);
+    }
+  };
+
+  const openAddModal = () => {
+    setForm(initialForm);
+    setEditingIndex(null);
+    setModalMode('add');
+    onOpen();
+  };
+
+  const handleEdit = (idx: number) => {
```

ESLint

[error] Replace `··` with `↹`

(prettier/prettier)